src/request.ts

```diff
@@ -18,12 +18,19 @@ export const request = async <T = Response>(
     retries: requestSettings.retries,
   }
 ): Promise<T> => {
-  const { userId, integrator, widgetVersion } =
+  const { userId, integrator, widgetVersion, apiKey } =
     ConfigService.getInstance().getConfig()
 
   options.retries = options.retries ?? requestSettings.retries
   try {
     if (!options.skipTrackingHeaders) {
+      if (apiKey) {
+        options.headers = {
+          ...options?.headers,
+          'X-LIFI-api-key': apiKey,
```

Comment:
Suurio

Reply:
Sry just my two cents: headers should be always lowercase, most of the client nowdays convert them to lowercase automatically- saying this because in the BE I had an issue while checking for a case sensitive header

Reply:
@mathiasmoeller @dnnnvx I'm happy with both ways as long as we stay consistent 😄 We can make them all lowercase 🙂

Reply:
@dnnnvx should we change to: 

- `x-lifi-sdk`
- `x-lifi-widget`
- `x-lifi-api-key`
- `x-lifi-userid` - this one would be cool to change to `x-lifi-user-id` if possible 🙂

Reply:
Yeah I think it's better, I worked on the backend context recently and all of the custom headers I had to check arrived in lowercase, so I'd say it's better to always use lowercase 👌

Reply:
I wouldn't rename existing headers. The lowercasing shouldn't matter that much, we can change that I guess